src/test/components/MarkerTable.test.js

```diff
@@ -16,29 +17,29 @@ import {
   changeMarkersSearchString,
 } from '../../actions/profile-view';
 import { ensureExists } from '../../utils/flow';
+import { getEmptyThread } from 'firefox-profiler/profile-logic/data-structures';
 
 import { storeWithProfile } from '../fixtures/stores';
 import {
-  getProfileWithMarkers,
+  getProfileFromTextSamples,
   getMarkerTableProfile,
+  addMarkersToThreadWithCorrespondingSamples,
 } from '../fixtures/profiles/processed-profile';
 import {
   getBoundingBox,
   fireFullClick,
   fireFullContextMenu,
 } from '../fixtures/utils';
 
+import type { CauseBacktrace } from 'firefox-profiler/types';
+
 describe('MarkerTable', function() {
-  function setup(markers) {
+  function setup(profile = getMarkerTableProfile()) {
```

Comment:
Instead of specifing markers, now we specify a full profile. Actually it was used only once before, to get a profile __without__ any marker, as you'll see below...